bookstore/clone.py

```diff
@@ -270,7 +270,7 @@ def build_content_model(self, content, target_path):
         dict
             Jupyter Contents API compatible model
         """
-        path = self.contents_manager.increment_filename(target_path)
+        path = self.contents_manager.increment_filename(target_path, insert='-')
```

Comment:
Seems unrelated, what was the reason for this?

Reply:
To create consistency between the S3 and FS clone APIs file name conflict policies. I can yank this out and make a separate PR if you'd like.

Reply:
No it's fine to leave in